patch typedef for `int64` which causes hang in `0602ResultantFanProjection`

cf:
https://github.com/void-linux/void-packages/pull/34182
https://trac.sagemath.org/ticket/32088
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=905300

--- a/src/vektor.h	2017-06-20 11:47:37.000000000 -0300
+++ b/src/vektor.h	2021-11-21 18:28:43.384750825 -0300
@@ -10,7 +10,7 @@
 
 using namespace std;
 
-typedef signed long int int64;
+typedef int64_t int64;
 
 void outOfRange(int i, int n);
 
